tokio/src/runtime/builder.rs

```diff
@@ -375,6 +385,30 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    #[cfg(feature = "blocking")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
+    /// Sets a custom timeout for a thread in the blocking pool.
+    /// By default, the timeout for a thread is constant and defined
+    /// in KEEP_ALIVE. In case the user wants to customize this
+    /// value, then they can use blocking_keep_alive() for it.
```

Comment:
I don't think "In case the user wants to customize this value, they can use `blocking_keep_alive()` for it" is adding anything to this documentation. I think it's pretty clear that this method can be used to customize the default value without this sentence.

I'd rather give an example of _why_ the user might want to override this value. But, if there aren't any obvious reasons, we should just remove this sentence, IMO.